src/modules/map/components/cesium/CesiumWMSLayer.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+    <div>
+        <slot />
+    </div>
+</template>
+
+<script>
+import { CoordinateSystem, WEBMERCATOR } from '@/utils/coordinateSystems'
+import { ImageryLayer, Rectangle, WebMapServiceImageryProvider } from 'cesium'
+import { WGS84_EXTENT } from '@/config'
+import { mapState } from 'vuex'
+import { YEAR_TO_DESCRIBE_ALL_OR_CURRENT_DATA } from '@/api/layers/LayerTimeConfigEntry.class'
+import GeoAdminWMSLayer from '@/api/layers/GeoAdminWMSLayer.class'
+import addImageryLayerMixins from '@/modules/map/components/cesium/utils/addImageryLayer-mixins'
+
+// todo should we adapt LOD to resolution?
+const MAXIMUM_LEVEL_OF_DETAILS = 18
+
+export default {
+    mixins: [addImageryLayerMixins],
+    props: {
+        wmsLayerConfig: {
+            type: [GeoAdminWMSLayer],
+            required: true,
+        },
+        previewYear: {
+            type: Number,
+            default: null,
+        },
+        projection: {
+            type: CoordinateSystem,
+            default: WEBMERCATOR.epsg,
+        },
+        zIndex: {
+            type: Number,
+            default: -1,
+        },
+    },
+    computed: {
+        ...mapState({
+            currentLang: (state) => state.i18n.lang,
+        }),
+        layerId() {
+            return this.wmsLayerConfig.geoAdminID || this.wmsLayerConfig.externalLayerId
+        },
+        opacity() {
+            return this.wmsLayerConfig.opacity || 1.0
+        },
+        wmsVersion() {
+            return this.wmsLayerConfig.wmsVersion || '1.3.0'
+        },
+        format() {
+            return this.wmsLayerConfig.format || 'png'
+        },
+        url() {
+            return this.wmsLayerConfig.getURL(this.previewYear, this.projection.epsgNumber)
```

Comment:
I think you can remove these two params, they were not removed when we worked on the preview year concept in the OL component 🙈